packages/front-end/components/Settings/EditDataSource/DataSourcePipeline/DataSourcePipeline.tsx

```diff
@@ -0,0 +1,102 @@
+import { CSSProperties, FC, ReactNode, useCallback, useState } from "react";
+import { FaPencilAlt } from "react-icons/fa";
+import usePermissions from "@/hooks/usePermissions";
+import { checkDatasourceProjectPermissions } from "@/services/datasources";
+import { DataSourceQueryEditingModalBaseProps } from "../types";
+import { EditDataSourcePipeline } from "./EditDataSourcePipeline";
+
+type DataSourcePipelineProps = DataSourceQueryEditingModalBaseProps;
+
+const DataSourcePipelineField: FC<{
+  title?: string;
+  style?: CSSProperties;
+  children?: ReactNode;
+  titleClassName?: string;
+}> = ({ children, title = "", titleClassName = "", style }) => {
+  return (
+    <div className={`mb-2 ma-5 ${titleClassName}`} style={style}>
+      {title}
+      {children}
+    </div>
+  );
+};
```

Comment:
Not worth creating a component for this.  Only used 3 times and 1 of them is unique (bold text).  Better to just do it all inline

Reply:
Agreed. It did more in an earlier rendition.